frappe/utils/html_utils.py

```diff
@@ -103,6 +103,15 @@ def get_icon_html(icon, small=False):
 	else:
 		return "<i class='{icon}'></i>".format(icon=icon)
 
+def unescape_html(value):
+	try:
```

Comment:
You can use `frrom six.moves.html_parser import HTMLParser`

It works on both 2 and 3